### DISCUSSION TITLE
How can ESP32 read RSSI from phone

### DISCUSSION THREAD

**Discussion by Tuong-FX16332** [h2zero/NimBLE-Arduino#405] [Q&A]
I am a newbie and I just start my first BLE project
In my project, ESP32 is server and my phone can connect with it 
How can ESP32 read RSSI after connected ?
Please help me  :<

**Comment by h2zero** [comment]
There is no method provided for this in the server code at this time, I will add this later. 

What you can do for now is use the low level functions for this:
```
    int8_t rssiValue = 0;
    ble_gap_conn_rssi(conn_id, &rssiValue); // conn_id can be obtained in the onConnect callback.
```

**Reply by Tuong-FX16332** [reply]
I did it 
thanks for your help
